### PR TITLE
Updated tests to use getLightblueFactory()

### PR DESCRIPTION
**Pull request by jewzaam**
Depends on https://github.com/lightblue-platform/lightblue-core/pull/355 being merged.

Local builds are failing but not because of anything I changed.

@dcrissman if this still fails here, might need your eyes on it.  It's complaining when trying to create metadata from test resource `metadata/datatype-metadata.json`.

### CODE REVIEW

**Comment by dcrissman** [comment]
@jewzaam an unintended result from some other changes elsewhere. #64 will fix it.

**Comment by jewzaam** [comment]
@dcrissman I merged that PR, merged master to this PR, and local builds still fail.

File: lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
         "objectClass": ["top", "groupofNames"],
         "cn": "#cn",
         "description": "#description",
-        "member": [#members]
+        "member": ["#members"]
     }
```
Comment (dcrissman) [inline lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json]: This change needs to be reverted. The test methods themselves are ensuring the quotation marks. I am fine if we want to change this, but lets do it in another issue/PR.

Comment (dcrissman) [inline lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json]: That, or I suppose the unit tests could be changed to not include the surrounding quotation marks.

Comment (jewzaam) [inline lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json]: Will revert, created https://github.com/lightblue-platform/lightblue-ldap/issues/65 to fix

**Comment by jewzaam** [comment]
Issues should be fixed now, local build is successful at least.. ready for review.

**Pull request merged**